.github/workflows/ci.yml

```diff
@@ -83,3 +84,11 @@ jobs:
       run: |
         stack test
 
+  nix-build-web:
```

Comment:
We should probably setup some sort of caching for this, but I don't necessarily think that should hold-up this PR from being merged.

Reply:
Note that Miso has it's own Cachix cache: https://github.com/dmjio/miso/blob/master/README.md#begin.